### PR TITLE
[release/content-build] Content Separation Release

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Getting a PR ready for the release of content build so that anyone working in files that have been moved will be required to get approval from someone who can let them know the file needs to be updated in another repo.

This is will remain in place until we get approval to remove the files completely.

**Tests that were moved to `content-build`**

[accessible-modal.e2e.spec.js](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/platform/site-wide/tests/accessible-modal.e2e.spec.js)

[system_home.cypress.spec.js](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/layouts/tests/vamc/system_home.cypress.spec.js)

[location_home.cypress.spec.js](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/layouts/tests/vamc/location_home.cypress.spec.js)

[00-required.e2e.spec.js](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/tests/home/00-required.e2e.spec.js)

### CODE REVIEW

**Review by jhonny218 — APPROVED** [review]

**Review by kelsonic — APPROVED** [review]
lgtm, could you post a link to the PR in content build where we add PW as codeowners to the template files?

**Review by mpelzsherman — APPROVED** [review]
Approving because I don't want to hold up the release.

However, for the record, I remain very concerned about the impact to developer workflow and potential breakages in production that this is going to create.

**Review by ncksllvn — APPROVED** [review]
LGTM, though I hope we will have VSP's dedicated support when/if unforeseen issues surface after deployments.

**Review by theodur — APPROVED** [review]

**Pull request merged**
